app/src/main/java/com/gh4a/utils/rx/RxTools.java

```diff
@@ -0,0 +1,106 @@
+package com.gh4a.utils.rx;
+
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+import io.reactivex.ObservableTransformer;
+import io.reactivex.functions.Action;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.DialogFragment;
+import android.view.View;
+import com.gh4a.BaseActivity;
+import com.gh4a.R;
+import com.gh4a.utils.UiUtils;
+import com.philosophicalhacker.lib.RxLoader;
+import java.util.concurrent.Callable;
+
+public class RxTools {
+    public static Observable runCallable(Callable call, BaseActivity activity, int loaderId, boolean refresh) {
+        return Observable.fromCallable(call)
```

Comment:
This shows `Unchecked assignment` warning here and in all the places that use this method. Adding generics should probably be enough to fix this. Like:

`public static <T> Observable<T> runCallbale(Callable<T> call, ...`